src/components/popups/InspectTask.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from "react";
+import "../../styles/popups/CreateTeam.scss";
+import { api, handleError } from "helpers/api";
+import PropTypes from "prop-types";
+import { Button } from "components/ui/Button";
+import { useNavigate, useParams } from "react-router-dom";
+
+const InspectTask = ({ isOpen, onClose, task }) => {
+  const [editMode, setEditMode] = useState(false);
+  const [taskTitle, setTaskTitle] = useState(task.title);
+  const [taskDescription, setTaskDescription] = useState(task.description);
+  const { teamId } = useParams();
+  const token = sessionStorage.getItem("token");
+  const [error, setError] = useState("");
+
+  if (!isOpen) return null;
+
+  const ActivateEditMode = () => {
+    setEditMode(true);
+  };
+  const DeactivateEditMode = () => {
+    setEditMode(false);
+  };
+
+  const doClose = () => {
+    setTaskTitle(task.title);
+    setTaskDescription(task.description);
+    setError("");
+    DeactivateEditMode();
+    onClose();
+  };
+
+  const EditTask = async () => {
+    try {
+      task.title = taskTitle;
+      task.description = taskDescription;
+      const requestBody = JSON.stringify(task);
+      const response = await api.put(
+        `/api/v1/teams/${teamId}/tasks/${task.taskId}`,
+        requestBody,
+        {
+          headers: {
+            Authorization: `${token}`,
+          },
+        }
+      );
+      //maybe remove when external api is ready
```

Comment:
This comment isn't clear to me as a reviewer? What all should be removed? Why only maybe?

Reply:
Refers to "location.reload();" right below the comment. Maybe remove when update via api